apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -70,24 +69,55 @@
           }}</span>
         </a>
       </div>
+      <mat-divider></mat-divider>
     </mat-nav-list>
-    <div
-      [ngClass]="isSmallScreen ? 'footer-mobile' : 'footer-desktop'"
-      [ngStyle]="{
-        padding: isSmallScreen
-          ? '0.5rem 1.5rem'
-          : isCollapsed
-            ? '1.5rem 1.6rem'
-            : '1.5rem 1.3rem',
-      }">
-      <button
-        mat-icon-button
-        id="logout"
-        class="material-icon-on-secondary-container"
-        [attr.aria-label]="'layout.logout' | translate"
-        (click)="logout()">
-        <mat-icon>logout</mat-icon>
-      </button>
+
+    <div class="bottom-section">
+      <mat-divider></mat-divider>
+      <div class="logout-container">
+        <button
+          mat-icon-button
+          id="logout"
+          class="material-icon-on-secondary-container"
+          [attr.aria-label]="'layout.logout' | translate"
+          (click)="logout()">
+          <mat-icon>logout</mat-icon>
+        </button>
+      </div>
+    </div>
+
+    <div *ngIf="!isCollapsed" class="footer">
+      <div class="info">
+        <a
+          [href]="'layout.impressum.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.impressum.text" | translate }}</a
+        >
+        <a
+          [href]="'layout.termsandconditions.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.termsandconditions.text" | translate }}</a
+        >
+        <a
+          [href]="'layout.dataprotection.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.dataprotection.text" | translate }}</a
+        >
+      </div>
+      <div class="version-information">
+        <a [routerLink]="'/gdpr'" rel="noopener">{{
+          "layout.gdpr" | translate
+        }}</a>
+        <span class="version-text"
+          >{{ "layout.version" | translate }}: {{ version }}</span
+        >
+      </div>
+    </div>
```

Comment:
Why should it not be possible to access these links, if the menu is collapsed or on a small screen? Not an expert, but there might be legal issues as a user does not have access to important documents.

So some way or another, these documents must still be accessible through the UI